lib/passageidentity/client.rb

```diff
@@ -54,6 +63,9 @@ def get_app
     # rubocop:enable Naming/AccessorMethodName
 
     # rubocop:disable Metrics/ParameterLists
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
nit: you can do this in 1 line.
 rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity

Reply:
[524f477](https://github.com/passageidentity/passage-ruby/pull/121/commits/524f47716b4be2a5c3a7512ad6ca6ece2dfe465f)